third_party/terraform/resources/resource_compute_network_peering.go.erb

```diff
@@ -7,7 +8,7 @@ import (
 
 	"github.com/hashicorp/terraform/helper/schema"
 	"google.golang.org/api/compute/v1"
-	"google.golang.org/api/googleapi"
```

Comment:
Line 173 depends on this import.

Reply:
I think my vscode did some automagic there, fixed it.